digests/validators/docker_test.go

```diff
@@ -0,0 +1,77 @@
+package validators
+
+import (
+	"testing"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/pivotal/image-relocation/pkg/image"
+	"github.com/pivotal/image-relocation/pkg/registry"
+	"github.com/stretchr/testify/assert"
+)
+
+type fakeClient struct {
+	copyFake       func(source image.Name, target image.Name) (image.Digest, int64, error)
+	digestFake     func(img image.Name) (image.Digest, error)
+	layoutFake     func(path string) (registry.Layout, error)
+	readLayoutFake func(path string) (registry.Layout, error)
+}
+
+func (f fakeClient) Copy(source image.Name, target image.Name) (image.Digest, int64, error) {
+	return f.copyFake(source, target)
+}
+
+func (f fakeClient) Digest(img image.Name) (image.Digest, error) {
+	return f.digestFake(img)
+}
+
+func (f fakeClient) NewLayout(path string) (registry.Layout, error) {
+	return f.layoutFake(path)
+}
+
+func (f fakeClient) ReadLayout(path string) (registry.Layout, error) {
+	return f.readLayoutFake(path)
+}
+
+func TestDigestMatches(t *testing.T) {
```

Comment:
Suggestion: this could be combined with the next test function to form a table test.